### DISCUSSION TITLE
Wrong CI test attached to my PR

### DISCUSSION THREAD

**Discussion by Ninzero** [Homebrew/discussions#4245] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.1-68-g85f9bb8
ORIGIN: https://github.com/Homebrew/brew
HEAD: 85f9bb84209d58f9331dae76b57c28f6c71a614b
Last commit: 8 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b38e16f2e8c488d9485a0f84eeb7dbf3631b5394
Core tap last commit: 8 hours ago
Core tap branch: master
Core tap JSON: 19 Feb 21:11 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_AUTO_UPDATE_SECS: 600
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: code -w
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_NO_AUTO_UPDATE: set
HOMEBREW_NO_INSTALL_CLEANUP: set
HOMEBREW_NO_INSTALL_FROM_API: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: dodeca-core 64-bit arm_blizzard_avalanche
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Not releated
```

### Description of issue

Three days ago, I opened a pull request https://github.com/Homebrew/homebrew-core/pull/123473 for a new formula [leetup](https://github.com/dragfire/leetup). This PR was closed because this project needed to be more noticeable, i.e. it did not reach the bare minimum of 75 stars (though it already had 73 stars at that time).

Yesterday I found out this project has reached the 75 stars standard, so I opened a new pull request https://github.com/Homebrew/homebrew-core/pull/123704. However, this PR was again closed by @SMillerDev because he believed there were some audits I ticked the box for that did not pass. I asked him to kindly point out which tests did not pass since I can promise that I have run each and every test I ticked box for, and they were all passed. But he did not reply to my comment.

My new PR also passed Github actions pull requests checks, as shown in this image:
![image](https://user-images.githubusercontent.com/20299935/220174077-a41baa79-769d-4b6f-ae68-329f514c9a1a.png)

However, I realized that there is also another GitHub action showing my PR did not pass some checks, as shown in this image:
<img width="692" alt="image" src="https://user-images.githubusercontent.com/20299935/220174478-1e459cc1-bc00-4ce0-8014-836eb16eef1a.png">

I suspect this is because I used the same commit as last time to create this PR. But this is a misunderstanding since these failed checks were created three days ago, and my PR was created yesterday.

I would be grateful if anyone could help reopen this pull request or explain which test I neglected.

**Comment by dtrodrigues** [comment, marked as answer]
I've reopened the PR and rebased to ensure a new run occurs that's unrelated to the previous run.

**Reply by Ninzero** [reply]
Thank you so much for the help. Really appreciate it.
